src/main/org/apache/tools/ant/taskdefs/optional/junitlauncher/confined/ListenerDefinition.java

```diff
@@ -86,6 +87,7 @@ public void setType(final ListenerType type) {
         switch (type.getValue()) {
             case LEGACY_PLAIN: {
                 this.setClassName("org.apache.tools.ant.taskdefs.optional.junitlauncher.LegacyPlainResultFormatter");
+                this.setExtension("xml");
```

Comment:
Given [line ](https://github.com/apache/ant/pull/168/commits/c1b2fa46c659425633668483f13a4064b965bb54#diff-7fa4f9478ca6c9afa377e09928e5f6718f712361d2f465629e031e479b61675fL289) I'd assume this should be "txt", not "xml"

Reply:
OK, given line 57, which defaults the extension to "txt", this xml should go to line 97/99 instead, no?

Reply:
Oops, that was a last second change. Fixed. Thanks for spotting this!